### ISSUE TITLE
Can't play songs from the MusicTime menu in VS Code

### ISSUE THREAD

**Issue by RickCoxDev** [swdotcom/swdc-vscode-musictime#1]
**Describe the bug**
```paste below
After logging in to spotify and selecting either web player or desktop player, I am not able to play any songs from the MusicTime menu in VS Code.
```
**ISSUE TYPE**
- Bug Report

**Steps to reproduce**

1. Go to MusicTime menu in VS Code
2. Verify that you are logged into Spotify and that you have either a Web or Dekstop player open
3. Make sure that either player is selected in MusicTime
4. Click any song in playlists or recommended.
5. Music doesn't play and there is no error thrown.

**Expected behavior**
```paste below
Music plays when a song is selected.
```

**ACTUAL RESULTS**

```paste below
Nothing plays when selected from the MusicTime menu. When a song is selected in Spotify the status bar displays correctly in VS Code and I'm able to stop/play or skip/go back. But selecting new songs still doesn't work.
```

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS Mojave v.10.14.6
 - Plugin version v.1.8.1

**Additional context**
```paste below

```

**Comment by brettmstevens7** [comment]
@RickCoxDev this happens on both the web player and desktop? Also do you have a premium or non-premium account? Think we might have an idea of what's going on. Will send an update soon.

**Comment by RickCoxDev** [comment]
I have a non-premium account and yes it happens for both web player and desktop.

**Comment by brettmstevens7** [comment]
@RickCoxDev we're working on a fix for this now that should be ready in the next release tomorrow.

**Comment by brettmstevens7** [comment]
@RickCoxDev This is fixed in 1.8.4. With a non-premium account on Mac, the player will only work with the desktop app installed (since we use applescript instead of Spotify APIs to control the player). Let me know if everything works on your end.

**Comment by RickCoxDev** [comment]
Everything seems to be working great now. Thanks!
